sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -1860,4 +1860,72 @@ class HiveDDLSuite
       }
     }
   }
+
+  Seq("PARQUET", "ORC", "TEXTFILE", "SEQUENCEFILE", "RCFILE", "AVRO").foreach { tableType =>
+    test(s"alter hive serde table add columns -- partitioned - $tableType") {
+      withTable("alter_add_partitioned") {
```

Comment:
The name is confusing. Let us just simplify it to `tab`. We already can know the scenario by the test case name.